src/net.h

```diff
@@ -874,6 +875,25 @@ class CNode
         assert(false);
     }
 
+    std::string ConnectionTypeToString() const {
```

Comment:
For new code it would make sense to properly clang-format it. See e.g. https://github.com/bitcoin/bitcoin/tree/master/contrib/devtools#clang-format-diffpy

Reply:
thanks, think I've updated all the commits with proper clang formatting. moving forward, will try to incorporate this into my workflow.